components/Player/PlayerHeader.css

```diff
@@ -24,10 +24,6 @@
   composes: image;
```

Comment:
I think you need to `rm -rf` your build directory to avoid adding the build files.